src/components/Input/style.tsx

```diff
@@ -0,0 +1,114 @@
+import React from "react";
+import styled, { StyledComponent } from "styled-components";
+
+import { InputProps } from "./type";
+
+export const StyledInput = styled.input`
+  flex: 1 1 20%;
+  border: none;
+  padding: 0;
+  width: 100%;
+  height: 100%;
+
+  color: #262626;
+  font-size: 0.875rem;
+  font-weight: 600;
+  line-height: 1.25rem;
+
+  &:focus {
+    outline: none;
+  }
+
+  &::placeholder {
+    color: #abaeb8;
+    font-weight: 500;
+  }
+
+  &:disabled {
+    background-color: "#F0F0F0";
```

Comment:
maybe change this into `background-color: #f0f0f0;` ?

Reply:
ok